homework.md

```diff
@@ -44,43 +44,42 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
-
+ Is a scripting or programming language that allows you to implement complex features on web pages
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
-
+When people want to show off code to each other they can put it on Github.
+Destroy others code. 
```

Comment:
I think I know what you mean, but instead of 'destroy' I'd go with making changes and building on others' work :) Also worth mentioning that Git and Github give you a timeline of changes made and you can go back to older versions if need be.